File: app/src/main/java/com/kavirelectronic/ali/kavir_info/Application.java
package com.kavirelectronic.ali.kavir_info;

import io.realm.Realm;
import io.realm.RealmConfiguration;
import uk.co.chrisjenx.calligraphy.CalligraphyConfig;

public class Application extends android.app.Application {
    @Override
    public void onCreate() {
        super.onCreate();
        Realm.init(this);
        RealmConfiguration config = new RealmConfiguration.Builder()
                .name("kavirDb.realm")
                .schemaVersion(1)
                .build();
        Realm.setDefaultConfiguration(config);

        CalligraphyConfig.initDefault(new CalligraphyConfig.Builder()
                .setDefaultFontPath("fonts/sans.ttf")
                .setFontAttrId(R.attr.fontPath)
                .build());

    }
}

File: app/src/main/java/com/kavirelectronic/ali/kavir_info/Application.kt
package com.kavirelectronic.ali.kavir_info

import android.app.Application
import io.realm.Realm
import io.realm.RealmConfiguration

class Application : Application() {
    override fun onCreate() {
        super.onCreate()
        Realm.init(this)
        val config = RealmConfiguration.Builder()
                .name("kavirDb.realm")
                .schemaVersion(2)
                .deleteRealmIfMigrationNeeded()
                .build()
        Realm.setDefaultConfiguration(config)
    }
}